src/JestExt.ts

```diff
@@ -349,14 +326,15 @@ export class JestExt {
     this.unknownItStyle = decorations.notRanItName()
   }
 
-  private shouldIgnoreOutput(text: string): boolean {
-    return text.includes('Watch Usage')
+  private shouldIgnoreOutput(_: string): boolean {
+    // this fails when snapshots change - to be revised - returning always false for now
+    // return text.includes('Watch Usage')
+    return false

```

Comment:
want to do anything about this?

Reply:
Yes. I work continuously with snapshots so I will keep eye on it. I have to look a bit closer on how output is being processed.

Reply:
No, no really progress on it. I was a bit idle waiting for this PR to be finished and merged. It takes very long time and it is a bit demotivating so I started to put my attention somewhere else for a moment. I also wasn't sure what will be the effect of some other investigations on how to communicate with Jest Process that was mentioned in some other discussions before. For this reason I did not rush up. I am less concerned with the console output, which I found buggy anyway - it seems to swallow lots of the contents because of this "Watch Usage" filtering? I was more concerned that Snapshots work. So maybe we keep snapshots working and we open the issue on filtering the output, which I think requires review anyway.

Reply:
> It takes very long time and it is a bit demotivating

Yea, sorry for the delay. Life got busy, and when I did find some time it wasn't enough to get through a PR of this size. It's even slower when you aren't familiar with the new or old code, and we don't have integration tests to make sure we've preserved behavior.
<br/>

> I am less concerned with the console output, which I found buggy anyway - it seems to swallow lots of the contents because of this "Watch Usage" filtering?

That's fair. I've reverted it and opened #228 for the filtering. We filter out the Watch Usage prompt since it's extra noise we don't need to see. It shouldn't swallow anything else.